src/libtilt/fft_utils.py

```diff
@@ -412,8 +412,15 @@ def _pad_to_best_fft_shape_2d(
     h, w = image.shape[-2:]
     ph, pw = fft_size_h - h, fft_size_w - w
     too_much_padding = ph > h or pw > w
-    padding_mode = 'reflect' if too_much_padding is False else 'constant'
-    image = F.pad(image, pad=(0, pw, 0, ph), mode=padding_mode)
+    if too_much_padding:
+        image_means = einops.reduce(
+            image, 'tilt h w -> tilt 1 1', reduction='mean'
```

Comment:
```suggestion
            image, '... h w -> ... 1 1', reduction='mean'
```

Making it able to deal with variable shapes :)